regolith/schemas.py

```diff
@@ -248,7 +248,9 @@
                               'Nuclear and Radiation Engineering '
                               'Program',
                     'end_year': 2011,
-                    'institution': 'The University of Texas at Austin',
+                    'group': 'ergs',
+                    'institution': 'columbiau',
```

Comment:
yes, its breaking that test because I blew away the name being matched in the test.

I can put it back as it was so it passes the test, and we can put in an issue to use our nice ``institutions`` colletion wherever possible in the cv and resume builders.